client/views/room/contextualBar/UserInfo/index.js

```diff
@@ -172,7 +172,7 @@ export const UserInfoWithData = React.memo(function UserInfoWithData({ uid, user
 		} = user;
 		return {
 			_id,
-			name: showRealNames ? name : username,
+			name,
```

Comment:
check the warning and the possibility of 'name' being ''

Reply:
Can you explain it more?

> check the warning and the possibility of 'name' being ''

Reply:
if the name is not provided ... we gonna see something like: 
![image](https://user-images.githubusercontent.com/5263975/112236512-733a2f80-8c1f-11eb-9fb4-60fc21ba0a23.png)

Reply:
So should we display username when the name is empty?

Reply:
Hello! Can you please help to solve this issue?